collections/nemo_nlp/nemo_nlp/data/datasets/bert_pretraining.py

```diff
@@ -71,13 +72,10 @@ def find_newlines(contents):
                             .replace(b"\xa0", b" ")
                         num_tokens = len(line.split())
 
-                        # Ensure the line has at least max_seq_length tokens
-                        if num_tokens >= max_seq_length:
-                            yield start - 1
-                            used_tokens += num_tokens
+                        yield new_start
 
-                        total_tokens += num_tokens
                         start = new_start + 1
+
```

Comment:
if we no longer track total_tokens value, we should remove it entirely. but i think we should still track that number of statistics.